includes/defaults.inc.php

```diff
@@ -947,3 +947,11 @@
 $config['api']['cors']['maxage'] = '86400';
 $config['api']['cors']['allowmethods'] = array('POST', 'GET', 'PUT', 'DELETE', 'PATCH');
 $config['api']['cors']['allowheaders'] = array('Origin', 'X-Requested-With', 'Content-Type', 'Accept', 'X-Auth-Token');
+
+// The target maximum hostname length when applying the shorthost() function.
```

Comment:
We need docs adding as well. If you can move all of these comments to https://github.com/librenms/librenms/blob/master/doc/Support/Configuration.md#webui-settings somewhere in that section that would be good.

After that, 👍 from me.